js/components/conceptset/included.html

```diff
@@ -10,14 +11,14 @@
       pageLength: tableOptions.pageLength,
       lengthMenu: tableOptions.lengthMenu,
       language: ko.i18n('datatable.language')
-    }"></faceted-datatable>
+    }"/>
```

Comment:
I would avoid this change:  KO can run into template parsing problems if you don't have explicit close tags.   Please avoid style changes unless it's absolutely necessary.